misc/extension_api_validation/4.5-stable.expected

```diff
@@ -99,3 +99,20 @@ Validate extension JSON: Error: Field 'builtin_classes/PackedVector3Array/method
 Validate extension JSON: Error: Field 'builtin_classes/PackedVector4Array/methods/duplicate': is_const changed value in new API, from false to true.
 
 Duplicate method made const. Compatibility methods registered.
+
+
+GH-112539
+---------
+
```

Comment:
```suggestion
```
For readability, we normally don't have an empty line here (with a few exceptions)